packages/keycodes/src/index.js

```diff
@@ -148,6 +143,17 @@ export const ZERO = 48;
 
 export { isAppleOS };
 
+/**
+ * Capitalise the first character of a string.
+ * @param {string} string String to capitalise.
+ * @return {string} Capitalised string.
+ */
+function capitaliseFirstCharacter( string ) {
```

Comment:
Minor language nit: since in web we generally go with the US conventions, I'd alter `capitalise` to `capitalize`.

```suggestion
function capitalizeFirstCharacter( string ) {
```

Reply:
Agreed. Although I personally favour the _s_ spelling, we should keep names consistent and greppable, @ellatrix

Reply:
Oh, looks like I forgot to fix it